benchmark/infrastructure/BenchmarkConfigurationShorthandVisitor.cpp

```diff
@@ -0,0 +1,65 @@
+// Copyright 2023, University of Freiburg,
+// Chair of Algorithms and Data Structures.
+// Author: Andre Schlegel (March of 2023, schlegea@informatik.uni-freiburg.de)
+
+#include "../benchmark/infrastructure/BenchmarkConfigurationShorthandVisitor.h"
+
+#include <any>
+
+#include "util/json.h"
+
```

Comment:
Please add (empty) comment lines for better visibility of function borders
`// _____________________________________________________`
(also in the header).

Reply:
Done, see [037a77a](https://github.com/ad-freiburg/qlever/pull/972/commits/037a77a43cef866c5211221b583ffaac15284885).